### ISSUE TITLE
Nested components 

### ISSUE THREAD

**Issue by seaj1one** [builderius/builderius#119]
I would love to be able to nest components inside each other. I'd like the flexility to build components from a button all the way upto a section.

**Comment by thisbit** [comment]
Whats stopping you, I do it all the time. That is kind of the point of components. Here is an entire video about this.
https://www.youtube.com/watch?v=oYszCJD59-Q&t=243s

**Comment by seaj1one** [comment]
I'm on holiday and I'll try and have a look at the link. I found I was unable to nest components within components.

**Comment by thisbit** [comment]
If you have some bugs, please report, components are nestable since probably when they came out ;)

**Comment by seaj1one** [comment]
I can report it as a bug for sure. I watched the video and see that the elements were stacked rather than neated. I'm happy to report a bug for sure.

**Comment by thisbit** [comment]
You really have to show it. I use this every day. I used it today. Production site. Bugs should be reported like this:

- what was the exact action you have performed?
- what was the result?
- what was the expected outcome?
- screenshots are welcome too

Thanks

**Comment by thisbit** [comment]
You really have to show it. I use this every day. I used it today. Production site.

**Comment by thisbit** [comment]
I will make an assumption as to what you may be doing from the way you phrased your last message. 

Are you trying to "nest" components the way you add a module inside another module? Like you take one component and drop it on top of another and expect it will go inside?

If yes, that is not how you do it. Module is just a single html element so one can drop another on it and get expected result. But component can have 20 html elements inside it, so you cannot drop another in it and expect to know where it lands.

- You nest a component inside another the same way you add component to a template.

- You add a "component module" to canvas and then select a desired component to go inside it.

- please watch this video and see what I am doing. Skip the intro and go to the build process, I show it all. https://builderius.io/using-components-to-improve-your-wordpress-development-workflow/

**Comment by seaj1one** [comment]
I made a short video where I created a div and a h3 as a component. I was looking to see if I could use a button component within the div and h3 component.  I could go to the div and h3 to add the button component. There is no problem. 

Sorry, there's no sound for some reason. 

https://youtu.be/Z8tzBtggewE

As I was recording the video I saw your last message. If components need to be stacked as best practice, that's ok. It's more curiosity, more than anything else. Sure thing I can take a look at the article, I probably missed that part.
